<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="user-Latitude-3440" failures="1" tests="8" name="Login" time="56.592" errors="0" timestamp="2025-07-22T22:58:48 IST">
  <testcase classname="testCases.TC_ContactUs" name="Tc_contactus" time="1.51"/>
  <testcase classname="testCases.TC_ContactUs" name="Tc_contactus" time="1.331"/>
  <testcase classname="testCases.TC_ContactUs" name="Tc_contactus" time="1.571"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="test_LoginDDT" time="11.625">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at testCases.TC_003_LoginDDT.test_LoginDDT(TC_003_LoginDDT.java:64)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- test_LoginDDT -->
  <testcase classname="testCases.TC_ContactUs" name="Tc_contactus" time="1.317"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="test_LoginDDT" time="11.75"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="test_LoginDDT" time="11.674"/>
  <testcase classname="testCases.TC_003_LoginDDT" name="test_LoginDDT" time="11.287"/>
</testsuite> <!-- Login -->
